src/components/capture/Create.tsx

```diff
@@ -32,24 +31,24 @@ function CaptureCreate() {
     const hasConnectors = connectorTags.length > 0;
 
     // Form store
-    const entityCreateStore = useRouteStore();
-    const messagePrefix = entityCreateStore(
+    const useEntityCreateStore = useRouteStore();
+    const messagePrefix = useEntityCreateStore(
```

Comment:
This change will be repeated in tons of files. Replaced the `entityCreateStore` with `useEntityCreateStore`